osf/models/schema_response.py

```diff
@@ -71,7 +71,8 @@ class SchemaResponse(ObjectIDMixin, BaseModel):
     class Meta:
         ordering = ['-created']
         indexes = [
-            models.Index(fields=['reviews_state'])
+            models.Index(fields=['reviews_state']),
+            models.Index(fields=['object_id', 'content_type'])
```

Comment:
swap the order on these? we should never be filtering on just object_id but may want to filter on just `content_type` (e.g. surfacing all SchemaResponses on Registrations)

Reply:
We're already indexing `content_type` -- it's a FK